ming_sim/cli_backend.py

```
@@ -1569,7 +1888,17 @@ def _run_extract() -> Dict[str, Any]:
                 captured = fut.result(timeout=timeout_s)
             finally:
                 pool.shutdown(wait=False, cancel_futures=True)
+    except UnknownParticipantEscalate as exc:
+        # 真不在册：通政司戏内回禀；禁吞 special_decree、禁除名照落。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)